src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/products.html.twig

```diff
@@ -119,6 +130,31 @@
                 </a>
               </td>
             {% endif %}
+
+            <td class="text-right">
+              <span class="partial-refund{% if product.isRefundable() == false %} hidden{% endif %}">
```

Comment:
Same here I think you can drop the `()` Twig is able to call getters on object automatically as if they are public fields as long as they respect the convention `get**()/is**()`